bin/seeds.rb

```diff
@@ -278,6 +280,15 @@
   "cinder_eqlx_use_chap"          => ['false'],
   "cinder_eqlx_chap_login"        => ['chapadmin'],
   "cinder_eqlx_chap_password"     => [SecureRandom.hex],
+  "cinder_dell_sc_san_ip"           => ['172.23.8.101'],
+  "cinder_dell_sc_san_login"        => ['Admin'],
+  "cinder_dell_sc_san_password"     => [SecureRandom.hex],
+  "cinder_dell_sc_iscsi_ip_address" => ['192.168.0.20'],
+  "cinder_dell_sc_iscsi_port"       => ['3260'],
+  "cinder_dell_sc_ssn"              => ['64702'],
+  "cinder_dell_sc_api_port"         => ['3033'],
+  "cinder_dell_sc_server_folder"    => ['server'],
+  "cinder_dell_sc_volume_folder"    => ['volume'],
```

Comment:
In the openstack docs website they are not openstack. Anyway these are defaults, don't think it matters
http://docs.openstack.org/kilo/config-reference/content/dell-storagecenter-driver.html